corehq/warehouse/models.py

```diff
@@ -62,3 +62,88 @@ def fail(self):
     )
     def decommission(self):
         return
+
+
+class BaseDim(models.Model):
+    domain = models.CharField(max_length=255)
+
```

Comment:
we might want to add an identifier to track which batch a dim record came from, but we can do that later when we figure out our etl strategy